### ISSUE TITLE
`depositValue` not defined in tutorial

### ISSUE THREAD

**Issue by cag** [gnosis/gnosis.js#109]
@Zupino noticed this:

In [this section](https://gnosisjs.readthedocs.io/en/latest/events-oracles-and-markets.html#events-and-collateral) of the tutorial, there is an example:

```js
async function buyAllOutcomes() {
    const txResults = await Promise.all([
        [gnosis.etherToken.constructor, await gnosis.etherToken.deposit.sendTransaction({ value: depositValue })],
        [gnosis.etherToken.constructor, await gnosis.etherToken.approve.sendTransaction(event.address, depositValue)],
        [event.constructor, await event.buyAllOutcomes.sendTransaction(depositValue)],
    ].map(([contract, txHash]) => contract.syncTransaction(txHash)))

    // Make sure everything worked
    const expectedEvents = [
        'Deposit',
        'Approval',
        'OutcomeTokenSetIssuance',
    ]
    txResults.forEach((txResult, i) => {
        Gnosis.requireEventFromTXResult(txResult, expectedEvents[i])
    })
}
buyAllOutcomes()
```

which is missing a definition for `depositValue` (should be 4 ETH following the tutorial text):

```js
const depositValue = 4e18
```

**Comment by srameshr** [comment]
@cag Also what does 
```
 [gnosis.etherToken.constructor, await gnosis.etherToken.deposit.sendTransaction({ value: depositValue })],
        [gnosis.etherToken.constructor, await gnosis.etherToken.approve.sendTransaction(event.address, depositValue)],
        [event.constructor, await event.buyAllOutcomes.sendTransaction(depositValue)],
``` do?

What is deposit, approve doing here? Assume intention of the user here is just to buy some tokens and nothing else.

**Comment by cag** [comment]
@srameshr This is a consequence of the way wrapped ether is specified and the way ERC20 tokens are specified respectively.

In short, `deposit` comes from the mechanics of [`W-ETH`](https://weth.io/). Basically, it's so ether can be used as an ERC20 token.

And `approve` comes from the [ERC20 standard](https://theethereum.wiki/w/index.php/ERC20_Token_Standard#Approve_And_TransferFrom_Token_Balance) itself. Referring to the linked article, you (`0x111...`) have to authorize the event contract (`0x222`) to perform a `transferFrom` [during the execution of](https://github.com/gnosis/gnosis-contracts/blob/91fb7c80d7105585726bf67c4f53290c00b7d897/contracts/Events/Event.sol#L57) `buyAllOutcomes`.

**Comment by cag** [comment]
Okay, the original issue should be addressed in the docs at #115 now. I will close this issue when that gets merged, and then I will open another issue wrt srameshr's comment
